### PR TITLE
Change Language ID to Locale class.

### PR DESCRIPTION
**Pull request by kitwtnb**
## Issue
- none

## Overview (Required)
- Language ID of String changes to Locale class.

## Links
- https://github.com/DroidKaigi/conference-app-2017/pull/242

This is different from this link content, but is it ok?
Or should I fix it in the same way?

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java [hunk 1]
```diff
@@ -38,12 +38,11 @@ public DroidKaigiClient(DroidKaigiService droidKaigiService, GithubService githu
...
     }
 
-    public Single<List<Session>> getSessions(@NonNull String languageId) {
-        switch (languageId) {
-            case LocaleUtil.LANG_JA:
-                return droidKaigiService.getSessionsJa();
-            default:
-                return droidKaigiService.getSessionsEn();
+    public Single<List<Session>> getSessions(@NonNull Locale locale) {
+        if (locale == Locale.JAPAN) {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java]: In my understanding, `Locale.JAPAN` represents country.
`Locale.JAPANESE` seems better 🤔 

- Locale.JAPAN => ja_JP
- Locale.JAPANESE => ja

Comment (kitwtnb) [inline app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java]: You are correct, I fixed it! 😄

**Comment by konifar** [comment]
Looks good 👍

**Comment by konifar** [comment]
Thank you for improvement!

**Pull request merged**